app/views/trainees/lead_schools/_results.html.erb

```diff
@@ -0,0 +1,31 @@
+<h1 class="govuk-heading-l govuk-!-margin-bottom-1">
+  <%= t("components.page_titles.search_schools.page_title_results") %>
+</h1>
+
+<div class="govuk-hint">
+  <%= t("components.page_titles.search_schools.sub_text_results") %> ‘<%= params[:query] %>’
+</div>
+
+<%= f.govuk_error_summary %>
+
+<%= f.govuk_radio_buttons_fieldset :lead_school_id, legend: { text: "" } do %>
```

Comment:
I think we need to have a non-blank legend for accessibility. Should we put the title inside the legend here?

Reply:
A non-blank legend for accessibility sounds good. Put it down on your ideas board and bring it up at the next opportunity?

I think putting the title inside the legend makes it more difficult to match the design requirements. I would prefer to keep the title at the top of the template.

I've just discovered we can use `legend: nil`instead of `legend: { text: "" }`. Nothing changes, but looks a little bit cleaner.

Reply:
Hmm yeah I see it will be annoying to get the error summary outside the fieldset but under the title 👍